Production/govuk_ios/ViewControllers/SearchViewController.swift

```diff
@@ -0,0 +1,131 @@
+import UIComponents
+
+class SearchViewController: BaseViewController,
+                            TrackableScreen {
+    private let viewModel: SearchViewModel
+
+    private lazy var searchBar: UISearchBar = {
+        let localSearchBar = UISearchBar()
+
+        localSearchBar.searchTextField.backgroundColor = UIColor.govUK.fills.surfaceSearchBox
+        localSearchBar.enablesReturnKeyAutomatically = false
+        localSearchBar.translatesAutoresizingMaskIntoConstraints = false
+        localSearchBar.placeholder = "Search"
+        localSearchBar.backgroundImage = UIImage()
+        localSearchBar.searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search",
+            attributes: [
+                NSAttributedString.Key.foregroundColor: UIColor.govUK.text.secondary,
+                NSAttributedString.Key.font: UIFont.govUK.body
+            ]
+        )
+        localSearchBar.searchTextField.leftView?.tintColor = UIColor.govUK.text.secondary
+        localSearchBar.searchTextField.addTarget(
+            self,
+            action: #selector(searchReturn),
+            for: UIControl.Event.editingDidEndOnExit
+        )
+
+        return localSearchBar
+    }()
+
+    var trackingName: String { "Search_Modal" }
+    var trackingTitle: String? { "Search" }
+
+    init(viewModel: SearchViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        if #available(iOS 15.0, *) {
+            let sheet = self.sheetPresentationController
+            sheet?.prefersGrabberVisible = true
+        }
+
+        configureUI()
+        configureConstraints()
+    }
+
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        searchBar.becomeFirstResponder()
+    }
+
+    private func configureUI() {
+        view.backgroundColor = GOVUKColors.fills.surfaceModal
+        configureNavBar()
+        view.addSubview(searchBar)
+    }
+
+    private func configureConstraints() {
+        NSLayoutConstraint.activate([
+            searchBar.topAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.topAnchor,
+                constant: -10
+            ),
+            searchBar.leftAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leftAnchor,
+                constant: 10
+            ),
+            searchBar.rightAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.rightAnchor,
+                constant: -10
+            ),
+            searchBar.heightAnchor.constraint(greaterThanOrEqualToConstant: 36)
+        ])
+    }
+
+    private func configureNavBar() {
+        self.title = "Search"
+        let navBarItem = self.navigationItem
+        let appearence = UINavigationBarAppearance()
```

Comment:
Is this consistent across the entire app? If so, might be worth adding a `UINavigationBarAppearance+GOVUK` extension like the `UITabBarAppearance+GOVUK` we have?